experimental/packages/opentelemetry-instrumentation-grpc/src/instrumentation.ts

```diff
@@ -26,9 +26,10 @@ import type {
   loadPackageDefinition,
   GrpcObject,
 } from '@grpc/grpc-js';
-
 import type * as grpcJs from '@grpc/grpc-js';
 
+import { getStringFromEnv } from '@opentelemetry/core';
```

Comment:
Note: this needs to change to the list, because it can (will?) have multiple values like db

Reply:
https://github.com/open-telemetry/opentelemetry-js/pull/5653/commits/e315dc0bfbbc07b78c58e136dbc75092869ea7e6